zilliqa/src/node.rs

```diff
@@ -777,6 +777,10 @@ impl Node {
         self.consensus.txpool_content()
     }
 
+    pub fn transaction_count_for_account(&self, account: Address) -> u64 {
+        self.consensus.transaction_count_for_account(account)
```

Comment:
Feel free to just make `transaction_pool` `pub` and use it directly - We do the same for other fields in `Consensus`.